workspaces/announcements/catalog-info.yaml

```diff
@@ -11,3 +11,18 @@ spec:
   type: website
   owner: john@example.com
   lifecycle: experimental
+
+---
```

Comment:
Not sure why this was added, did the CI say it was needed?

Reply:
No, we need to remove this duplicate. Thanks for surfacing.